packages/nextjs/app/builders/page.tsx

```diff
@@ -61,41 +52,101 @@ const Builders: NextPage = () => {
     receiptData: true,
   });
 
+  const { data: checkedInCounter } = useScaffoldReadContract({
+    contractName: "BatchRegistry",
+    functionName: "checkedInCounter",
+  });
+
   useEffect(() => {
     if (!isLoadingEvents && events != undefined) {
-      setBuildersCheckedIn(events.length);
       setBuilders(events.map(e => getBuilderFromEvent(e)));
+      setIsLoading(false);
     }
   }, [isLoadingEvents, events]);
 
   return (
-    <>
-      <div className="text-center bg-secondary p-10">
-        <h1 className="text-4xl my-0">Batch #9 people 💻 </h1>
+    <div className="dark:bg-zinc-950 bg-zinc-200 dark:text-st_white text-st_background min-h-screen h-fit relative overflow-clip flex justify-center">
+      <ShootingStars />
+      <StarsBackground />
+
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image
+          src="/blur_yellow.svg"
+          alt="Blurred Yellow Background"
+          className="absolute -top-[10%] -left-[40%] w-200 h-200 blur-2xl"
+          width={2000}
+          height={2000}
+        />
       </div>
-      <div className="max-w-screen-lg mx-auto mt-10 p-6 bg-base-300 shadow-md rounded-lg">
-        <div className="container mx-auto px-4 py-8">
-          <div className="mb-12">
-            <h2 className="text-2xl font-semibold text-center mb-6">Mentors</h2>
-            <div className="grid grid-cols-3 gap-6">
-              {mentors.map((mentor, i) => (
-                <MentorPicture person={mentor} key={i} />
-              ))}
+
+      <div className="absolute w-full h-full pointer-events-none">
+        <Image
+          src="/blur_blue.svg"
+          alt="Blurred Blue Background"
+          className="absolute -bottom-[10%] -right-[40%] blur-2xl"
+          width={2000}
+          height={2000}
+        />
+      </div>
+
+      <div className="flex flex-col gap-12 max-w-7xl justify-between w-full mt-10 sm:mt-18 z-10">
+        {/* Mentor Section */}
+        <div className="flex flex-col px-10 gap-4 w-full ">
+          {/* Mentor Header */}
+          <div className="flex gap-2 sm:gap-5 items-center justify-between">
+            <div className="tracking-wider flex flex-col items-start">
+              <h1 className="font-semibold text-xl sm:text-2xl md:text-3xl">Our Mentors..</h1>
```

Comment:
If I reduce the screen size, the left card becomes barely visible, and I'm unable to scroll further to the left.

![Screenshot 2024-09-26 at 19 27 14](https://github.com/user-attachments/assets/540773a7-e2f0-4559-9260-84c230d8e70c)

Reply:
fixed